setup.py

```diff
@@ -1,15 +1,65 @@
-import setuptools
-import re
+# -*- coding: utf-8 -*-
+
+import setuptools, versioneer
+
+# Package Metadata
+# ----------------
+#
+# Normally we'd have a `pds` namespace package with `doi_service` inside of it, but when I try to do that
+# I get errors like:
+#
+#    connexion.exceptions.ResolverError: <ResolverError: Cannot resolve operationId
+#    "pds_doi_service.api.controllers.dois_controller.get_dois"!
```

Comment:
This happened because any changes in the location of the API modules needs to be reflected within the `swagger.yaml` definition file that gets fed to `connexion` when launching the API.

Peep the `x-openapi-router-controller` fields within https://github.com/NASA-PDS/pds-doi-service/blob/main/pds_doi_service/api/swagger/swagger.yaml if you're still interested in making this namespace change.

Reply:
Ahhhhhh. Okay, hard pass on the namespace (reason: sprint deadline).